### PR TITLE
Issue #7620: Update doc for FileTabCharacter

### PR DESCRIPTION
**Pull request by Rainy0707**
Fixes Issue #7620 
Capture_filetab

$ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="FileTabCharacter">
        <property name="eachLine" value="true"/>
    </module>
</module>

$ cat Test.java
public class Test {
int i;

public void myTest() {
        for(int i=0; i<10; i++){
        }

for(int i=0; i<10; i++){
}
}

}

$ java -jar checkstyle-8.30-all.jar -c config.xml Test.java
Starting audit...
[ERROR] C:\Users\1\Desktop\New_folder\Test.java:5:5: Line contains a tab character. [FileTabCharacter]
[ERROR] C:\Users\1\Desktop\New_folder\Test.java:6:7: Line contains a tab character. [FileTabCharacter]
Audit done.
Checkstyle ends with 2 errors.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java [hunk 1]
```diff
@@ -68,6 +68,18 @@
  *   &lt;property name=&quot;eachLine&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>violation</p>
+ * <pre>
+ * for(int i=0; i&lt;10; i++){
+ *     //violation(extra spaces after open bracket)
```
Comment (esilkensen) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java:74]: this comment seems to refer to a different check (spaces after open bracket) and not tab characters

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java [hunk 2]
```diff
@@ -68,6 +68,18 @@
...
+ * }
+ * </pre>
+ * <p>acceptable code</p>
+ * <pre>
+ * for(int i=0; i&lt;10; i++){
+ *     //OK(no space after bracket)
```
Comment (esilkensen) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java:80]: this comment seems to refer to a different check (spaces after open bracket) and not tab characters

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java (hunk 1 above)
Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java:74]: it is very hard to show something that is not visible so example of code should be in form like:
```
$ cat Test1.java 
public class Test1 {
	int field; // tab character at the beginning of line
}

$ cat -A Test1.java 
public class Test1 {$
^Iint field; // tab character at the beginning of line$
}$
```

**Comment by romani** [comment]
abandoned and incomplete

**Pull request closed without merging**